lib/rules/cli/4.1.yaml

```diff
@@ -1150,11 +1150,13 @@
     :type: --type=<value>
     :whitelist: --whitelist=<value>
 :oadm_inspect:
-  :cmd: oc adm inspect <resource_type> <resource_name>
+  :cmd: oc adm inspect
   :options:
     :A: --all-namespaces=<value>
     :all_namespaces: --all-namespaces=<value>
     :dest_dir: --dest-dir=<value>
+    :resource_type: <value>
+    :resource_name: <value>
```

Comment:
Sort them in alphanumeric order.

Reply:
@liangxia  Updated , thanks, please help review and merge.